### PR TITLE
Feat/cms

### PR DESCRIPTION
**Pull request by xandergroeneveld**
New homepage
Installed CMS

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -36,6 +36,7 @@ gem "simple_form"
 gem "country_select"
 gem "faker"
 gem "stripe"
+gem 'comfortable_mexican_sofa', '~> 1.12.0'
```
Comment (groeneveld91) [inline Gemfile]: "

File: app/assets/javascripts/comfy/admin/cms/custom.js.coffee [hunk 2]
```diff
@@ -0,0 +1 @@
+# Custom JS for the admin area
```
Comment (groeneveld91) [inline app/assets/javascripts/comfy/admin/cms/custom.js.coffee]: prob forgot to delete this file?

File: app/assets/javascripts/pages/all.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+$(document).on('turbolinks:load', function(){
+  $('.ui.accordion')
```
Comment (groeneveld91) [inline app/assets/javascripts/pages/all.js]: new style.. 3 lines ?

File: app/assets/stylesheets/comfy/admin/cms/custom.sass [hunk 4]
```diff
@@ -0,0 +1 @@
+// custom CSS for admin area
```
Comment (groeneveld91) [inline app/assets/stylesheets/comfy/admin/cms/custom.sass]: delete ?

File: app/views/includes/_mobile_menu.html.erb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<div id="mobile-menu" class="ui container">
+  <div class="ui large secondary inverted pointing menu">
+    <%= image_tag "header-logo-edit-2.png", class: "header item" %>
+    <a class="toc item">
+      <i class="sidebar icon"></i>
+    </a>
+    <%= render partial: "includes/menu_items", locals: {sidebar: false} %>
```
Comment (groeneveld91) [inline app/views/includes/_mobile_menu.html.erb]: nice i should start using this locals option

File: config/initializers/comfortable_mexican_sofa.rb [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+  # Reveal partials that can be overwritten in the admin area.
+  # Default is false.
+  #   config.reveal_cms_partials = false
+
+end
+
+# Default credentials for ComfortableMexicanSofa::AccessControl::AdminAuthentication
+# YOU REALLY WANT TO CHANGE THIS BEFORE PUTTING YOUR SITE LIVE
+if Rails.env.production?
+  ComfortableMexicanSofa::AccessControl::AdminAuthentication.username = 'admin'
```
Comment (groeneveld91) [inline config/initializers/comfortable_mexican_sofa.rb]: Should this be in version control or ENV variables ?

File: db/cms_fixtures/sample-site/categories/files.yml [hunk 7]
```diff
@@ -0,0 +1 @@
+- File Category
```
Comment (groeneveld91) [inline db/cms_fixtures/sample-site/categories/files.yml]: can we clean out?

File: db/cms_fixtures/sample-site/categories/pages.yml [hunk 8]
```diff
@@ -0,0 +1 @@
+- Page Category
```
Comment (groeneveld91) [inline db/cms_fixtures/sample-site/categories/pages.yml]: can we clean out?

File: db/cms_fixtures/sample-site/categories/snippets.yml [hunk 9]
```diff
@@ -0,0 +1 @@
+- Snippet Category
```
Comment (groeneveld91) [inline db/cms_fixtures/sample-site/categories/snippets.yml]: can we clean out?

File: db/cms_fixtures/sample-site/files/_sample.jpg.yml [hunk 10]
```diff
@@ -0,0 +1,5 @@
+label: Fixture File
```
Comment (groeneveld91) [inline db/cms_fixtures/sample-site/files/_sample.jpg.yml]: can we clean out?

File: db/cms_fixtures/sample-site/layouts/default/attributes.yml [hunk 11]
```diff
@@ -0,0 +1 @@
+label: Default Fixture Layout
```
Comment (groeneveld91) [inline db/cms_fixtures/sample-site/layouts/default/attributes.yml]: Are all of these `db/cms_fixtures/sample-site` files being used or can we clean out ?

File: db/seeds.rb [hunk 12]
```diff
@@ -9,7 +9,9 @@
   require_relative "./seeds/workers"
   require_relative "./seeds/farmers"
 when "staging"
-  # Add staging specific seeds
+  require_relative "./seeds/admins"
```
Comment (groeneveld91) [inline db/seeds.rb]: awesome. happy with this setup?

File: db/migrate/20160913083624_create_cms.rb [hunk 13]
```diff
@@ -0,0 +1,133 @@
...
+    create_table :comfy_cms_categorizations, :force => true do |t|
+      t.integer :category_id,       :null => false
+      t.string  :categorized_type,  :null => false
+      t.integer :categorized_id,    :null => false
+    end
+    add_index :comfy_cms_categorizations, [:category_id, :categorized_type, :categorized_id], :unique => true,
+      :name => 'index_cms_categorizations_on_cat_id_and_catd_type_and_catd_id'
+  end
+
+  def self.down
```
Comment (groeneveld91) [inline db/migrate/20160913083624_create_cms.rb]: that's a lot of tables!

**Comment by groeneveld91** [comment]
Awesome looks good. Biggest thing was prob just cleaning out `db/cms_fixtures/sample-site/` if it is not needed. 
***Also putting the cms login creds in ENV variables, taking them out of version control***

File: config/initializers/comfortable_mexican_sofa.rb (hunk 6 above)
Comment (xandergroeneveld) [inline config/initializers/comfortable_mexican_sofa.rb]: yep

**Pull request closed without merging**